docs/gno-tooling/cli/gnokey/full-security-tx.md

```diff
@@ -0,0 +1,128 @@
+---
+id: full-security-tx
+---
+
+# Making an airgapped transaction
+
+## Prerequisites
+
+- **`gnokey` installed.** Reference the
+  [Local Setup](../../../getting-started/local-setup/installation.md#2-installing-the-required-tools) guide for steps
+
+## Overview
+
+`gnokey` provides a way to create a transaction, sign it, and later
+broadcast it to a chain in the most secure fashion. This approach, while more 
+complicated, grants full control and provides airgap support.
+
+The indented purpose of this functionality is to provide maximum security when 
+signing and broadcasting a transaction. In practice, this procedure should take
+place on two separate machines, one with access to the internet (`Machine 1`), 
+and the other one without (`Machine 2`), with the separation of steps as follows:
+1. `Machine 1`: Fetch account information from the chain
+2. `Machine 2`: Create an unsigned transaction locally
+3. `Machine 2`: Sign the transaction
+4. `Machine 1`: Broadcast the transaction
+
+For the sake of simplicity, in this example, we will assume that the procedure 
+is happening on two machines, and we will again use the Userbook 
+realm on the Portal Loop testnet.
+
+## Fetching account information from the chain
+
+First, we need to fetch data for the account we are using to sign the transaction,
+using the [auth/accounts](./querying-a-network.md#authaccounts) query:
```

Comment:
It might be cool to clarify that this operation takes place on `Machine 1` even though there’s a list of steps above.

Reply:
And similar to what I mentioned above, I think it’s better to explain what info we need to fetch (`account number` and `sequence`) before explaining how to do it. Personally, reading the paragraph with the idea that we’re going to fetch some "data" without knowing what it is makes it a bit confusing for me.

Reply:
I added an extra explanation right below the paragraph, explaining why you need to fetch these. [e24eabd](https://github.com/gnolang/gno/pull/2226/commits/e24eabde12ab64e1ca64e32bc2c8d9c013210fe6)

This is another spot for a glossary doc that will explain what these are and what they're for, for sure :)

Reply:
A glossary document could be very nice indeed 👍